client/my-sites/stats/stats-subscribers-page/subsriber-highlights.jsx

```diff
@@ -0,0 +1,97 @@
+import { HighlightCard, Popover } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import '../wordads/highlights-section.scss';
+
+function getSubscriberHighlights() {
+	const highlights = [
+		{
+			id: 1,
+			heading: 'Total email subscribers',
+			count: 1410,
+			previousCount: 1080,
+		},
+		{
+			id: 2,
+			heading: 'Free email subscribers',
+			count: 1340,
+			previousCount: 1034,
+		},
+		{
+			id: 3,
+			heading: 'Paid email subscribers',
+			count: 70,
+			previousCount: 46,
+		},
+		{
+			id: 4,
+			heading: 'WordPress.com subscribers',
+			count: 322,
+			previousCount: 326,
+		},
+	];
+	return highlights;
+}
+
+function SubscriberHighlightsHeader() {
+	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
+	const infoReferenceElement = useRef( null );
+	const localizedTitle = translate( 'All time stats', {
+		comment: 'Heading for Subscribers page highlights section',
+	} );
+	const showNotices = true;
+	const tooltipContent = <p>Tooltip content goes here.</p>;
+	return (
+		<h1 className="highlight-cards-heading">
```

Comment:
Could we add a subscriber-section-specific class name here?

Reply:
Will do. I was looking at the various section headers across the Traffic, Insights, and Ads pages and they all use this same CSS class except for the chart header and the "Posting activity" header. I think it makes sense to review this and pull it into a mix-in or standalone "section header" component.